### PR TITLE
Fixed bug with crash

### PR DESCRIPTION
**Pull request by shamilovstas**

### CODE REVIEW

File: android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package ir.sohreco.androidfilechooser;
 
-public class ExternalStorageNotAvailableException extends Exception {
-    public ExternalStorageNotAvailableException() {
+class ExternalStorageNotAvailableException extends RuntimeException {
```
Comment (MostafaNasiri) [inline android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java]: So why did you do this? Programmer must handle the situation of not having an external storage.

Comment (shamilovstas) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java]: I think that the idea of checked exception i n java is not an ideal. If user of this class not sure that external storage is available, then he must check it before using file picker dialog. But it just my opinion, if you sure that this exception must be checked, we can discuss it.

Comment (MostafaNasiri) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java]: What you're saying is ideal but many developers don't think about various situations. So I think that forcing the developer to handle this situation decreases the chance of their app getting crashed. Thank you for your attention.

File: android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java [hunk 2]
```diff
@@ -49,7 +48,7 @@
         void onSelect(String path);
     }
 
-    public enum ChooserType {
+    public enum ChooserType implements Serializable {
```
Comment (MostafaNasiri) [inline android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java]: Enums are inherently Serializable. Aren't they?

Comment (shamilovstas) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/FileChooserDialog.java]: You are totally right. I'll fix it in next commit.

File: android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java (hunk 1 above)
Comment (shamilovstas) [reply android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java]: Maybe, you're right. Okay, i'll change it.

**Pull request merged**